Makefile

```diff
@@ -54,11 +54,11 @@ test-e2e:
 
 .PHONY: test-e2e-postdeploy
 test-e2e-postdeploy:
-	go test -timeout 0 -count=1 ./test/e2e/postdeploy/...
+	go test -v -timeout 0 -count=1 ./test/e2e/postdeploy/...
```

Comment:
This will fill up the ci e2e logs with test output for successful tests, which is generally not what we want.

Reply:
I don't think it's going to actually fill anything, just add some context but I'm fine with removing and going back to only on failures.